### PR TITLE
Fix new clippy lints

### PR DESCRIPTION
**Pull request by thibault-martinez**
## Change checklist

Tick the boxes that are relevant to your changes, and delete any items that are not.

- [ ] I have followed the contribution guidelines for this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that new and existing unit tests pass locally with my changes

### CODE REVIEW

File: types/tests/rent.rs [hunk 1]
```diff
@@ -17,12 +17,7 @@ fn config() -> RentStructure {
 
 fn output_in_range(output: Output, range: std::ops::RangeInclusive<u64>) {
     let cost = output.rent_cost(&config());
-    assert!(
-        range.contains(&cost),
-        "{:#?} has a required byte cost of {}",
-        output,
-        cost
-    );
+    assert!(range.contains(&cost), "{output:#?} has a required byte cost of {cost}",);
```
Comment (Thoralf-M) [inline types/tests/rent.rs]:
```suggestion
    assert!(range.contains(&cost), "{output:#?} has a required byte cost of {cost}");
```

Comment (thibault-martinez) [reply types/tests/rent.rs]: I wish fmt would get rid of these :(

Comment (Thoralf-M) [reply types/tests/rent.rs]: Open a PR to rustfmt? :trollface:

Comment (thibault-martinez) [reply types/tests/rent.rs]: :weirdsmileguy:

**Review by Thoralf-M — APPROVED** [review]

**Review by samuel-rufi — APPROVED** [review]

**Pull request merged**